docs/app.md

```diff
@@ -264,42 +283,49 @@ A task that is completed has a check-mark `✔`. Tasks that are still todo will
 
 Additionally, after some trial time has passed, we show a link to a contact form where you can ask us to extend the trial time. Here you'll write us something about your goals with the trial and we are happy to extend the trial for much longer period. You can see those tasks in the Dashboard with the overview of your App.
 
+
 ### Upgrading the trial
 
 You can, of course, upgrade from your trial to a paid plan at any time. You can do so under your Apps overview.
 
+
 ### The end of a trial
 
 When the trial is finally over, the trial App will be deleted. That might sounds cruel, but please consider that: We will not keep your data hostage until you pay us. 
 
 Good news is: we will inform you by e-mail before it happens and you can start a new trial right away, after your current trial ended. In many cases you can easily re-deploy your code base to the new App: When deploying with [Git](git-deployment), you just change the remote and push again. When using our CMS/framework presets with predefined [ENV vars](env-vars) database access will work without code modification.
 
 
+## Resetting an App
+
+To reset an App, visit the App in the Dashboard, find and hit the "Reset App" button. This will restart the Apache service and the PHP-FPM processes - nothing more. There will be a short downtime of the App. This can help in certain edge cases, such as persistent 408 errors or hanging PHP processes with 503 errors. Use carefully, it should not be used on a regular basis. Please <a href="javascript:Intercom('showNewMessage')">contact us</a>. if you have ongoing trouble with your App.
```

Comment:
Should be a comma (,) instead (.) after "contact us".

Reply:
Good catch.